client/ios/dropin/Components/EventFormView.swift

```diff
@@ -0,0 +1,237 @@
+        }
+    }
+    
+    var body: some View {
+        NavigationStack {
+            Form {
+                Section {
+                    TextField("Title", text: $event.title)
+                        .font(.title)
+                        .focused($focusedField, equals: .title)
+                    
+                    TextField("Description", text: $event.description)
+                        .focused($focusedField, equals: .description)
+                } header: {
+                    Text("What's happening?")
+                        .font(.headline)
+                        .foregroundColor(.black)
+                        .textCase(nil)
+                }
+                
+                Section {
+                    HStack {
+                        Text("Start")
+                            .frame(width: 80, alignment: .leading)
+                        
+                        Spacer()
+                        
+                        DatePicker("", selection: $event.start, displayedComponents: [.hourAndMinute])
+                            .labelsHidden()
+                            .onChange(of: event.start) { oldValue, newValue in
+                                if newValue >= event.end {
+                                    event.end = Calendar.current.date(
+                                        byAdding: .minute,
+                                        value: 90,
+                                        to: newValue
+                                    ) ?? event.end
+                                }
+                            }
+                    }
+                    
+                    HStack {
+                        Text("End")
+                            .frame(width: 80, alignment: .leading)
+                        
+                        Spacer()
+                        
+                        DatePicker("", selection: $event.end, displayedComponents: [.hourAndMinute])
+                            .labelsHidden()
+                            .onChange(of: event.end) { oldValue, newValue in
+                                if newValue <= event.start {
+                                    event.end = Calendar.current.date(
+                                        byAdding: .minute,
+                                        value: 90,
+                                        to: event.start
+                                    ) ?? event.end
+                                }
+                            }
+                    }
+                } header: {
+                    Text("When?")
+                        .font(.headline)
+                        .foregroundColor(.black)
+                        .textCase(nil)
+                }
+                
+                Section {
+                    VStack {
+                        HStack {
+                            Image(systemName: "magnifyingglass")
+                                .foregroundColor(.gray)
```

Comment:
icon is set too far to the right and not symetrically padded.
would add a .padding(.leading)
```suggestion
                            Image(systemName: "magnifyingglass")
                                .foregroundColor(.gray)
                                .padding(.leading)
```